spglm/base.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, model, params, **kwd):
         self.initialize(model, params, **kwd)
         self._data_attr = []
 
-    def initialize(self, model, params, **kwd):
```

Comment:
Tricky change as until now, passing additional kwargs did not raise but now it will. I would revert it.

Reply:
Done.